tests/YGMeasureTest.cpp

```diff
@@ -21,8 +21,24 @@ static YGSize _measure(YGNodeRef node,
   }
 
   return YGSize {
-      .width = 10,
-      .height = 10,
+      width = 10,
```

Comment:
It seems the problem here is, that we compile with c++ (```*.cpp```) and there is no c99 compliance for c++ compilation on VS15. 😞